nameable.rb

```diff
@@ -0,0 +1,24 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+end
```

Comment:
- [ ] As per the requirements, I believe you forgot to Implement the correct_name method that returns the result of the correct_name method of the @namable.

See example:
```ruby
def correct_name
   @nameable.correct_name
end
```

Reply:
Yes, thank you. I forgot put it.